tmpl/wiki/faq.md

```diff
@@ -1,25 +1,24 @@
+MirageOS is a '[library operating system](http://anil.recoil.org/papers/2013-asplos-mirage.pdf)', which means that it can run on any target for which a suitable bootloader and drivers exist. The most mature targets are Unix and Xen.  Support for KVM via [solo5](/blog/introducing-solo5) is also fairly well-supported.  We have prototypes that compile the same application source code (e.g. the MirageOS website) to run as kernel modules inside FreeBSD, or even to JavaScript. MirageOS provides support to more easily target such diverse environments due to its emphasis on modular programming and compile-time specialisation.  If you'd like to dig into some code, see [the mirage-platform library](https://github.com/mirage/mirage-platform), which has the OCaml modules referred to in the Unix and Xen targets.
 
+*Q: Is MirageOS 'production ready'?*
 
-*Q: Is MirageOS 1.0 'production ready'*
-
-The 1.0 release is the first 'stable toolkit' release that is sufficient to self-host its infrastructure on the Internet. It is a baseline for the early adopters of MirageOS to integrate it into their own products, such as XenServer (whose management stack is written in OCaml and already uses some of the MirageOS libraries). We are also conducting research into novel databases, filesystems and security technologies at Cambridge that use MirageOS as the baseline to build complex distributed systems. The 1.0 release is still missing some significant functionality (such as SSL support), but development is proceeding fast to plug these gaps in 2014.
-
-More immediately, and in addition to the XenServer integration, members of the team (and we hope others) will be using MirageOS to create and manage their own VMs for websites and blogs!
+The 1.0 release is the first 'stable toolkit' release that is sufficient to self-host its infrastructure on the Internet.  [The current release](https://github.com/mirage/mirage/releases/latest) includes many improvements to stability and more supported features, including support for [gathering entropy](/blog/mirage-entropy), [execution trace collection and visualisation](http://roscidus.com/blog/blog/2014/10/27/visualising-an-asynchronous-monad/), and [ARM](https://mirage.io/blog/introducing-xen-minios-arm).  The various libraries used in MirageOS (e.g. [mirage-tcpip](https://github.com/mirage/mirage-tcpip) and [Irmin](https://github.com/mirage/irmin)) follow their own release cadences, but similarly are in use in self-hosting infrastructure.  Some libraries are currently being used in large-scale commercial products like [Docker for Mac and Windows](https://mirage.io/blog/introducing-xen-minios-arm).
 
+We are also conducting research into novel databases, filesystems and security technologies at Cambridge that use MirageOS as the baseline to build complex distributed systems.
```

Comment:
On rereading this, I suspect it refers to Irmin and... ?  I'm going to just remove it, but if anyone has a strong opinion about what should be concretely referred to, let me know.